test/ZNSAddressResolver.test.ts

```diff
@@ -1,195 +1,232 @@
     await expect(
-      znsAddressResolver.connect(operator).setAccessController(operator.address)
+      zns.addressResolver.connect(operator).setAccessController(operator.address)
     ).to.be.revertedWith(
       getAccessRevertMsg(operator.address, ADMIN_ROLE)
     );
   });
 
   it("Should not allow non-owner address to setAddress", async () => {
     await expect(
-      znsAddressResolver.connect(addr1).setAddress(wilderDomainNameHash, addr1.address)
-    ).to.be.revertedWith(
-      "ZNSAddressResolver: Not authorized for this domain"
-    );
+      zns.addressResolver.connect(user).setAddress(wilderDomainHash, user.address)
+    ).to.be.revertedWith("ZNSAddressResolver: Not authorized for this domain");
   });
 
   it("Should allow owner to setAddress and emit event", async () => {
     await expect(
-      znsAddressResolver.connect(owner)
-        .setAddress(wilderDomainNameHash, addr1.address)
+      zns.addressResolver.connect(deployer)
+        .setAddress(wilderDomainHash, user.address)
     )
-      .to.emit(znsAddressResolver, "AddressSet")
-      .withArgs(wilderDomainNameHash, addr1.address);
+      .to.emit(zns.addressResolver, "AddressSet")
+      .withArgs(wilderDomainHash, user.address);
 
-    const resolvedAddress = await znsAddressResolver.getAddress(wilderDomainNameHash);
-    expect(resolvedAddress).to.equal(addr1.address);
+    const resolvedAddress = await zns.addressResolver.getAddress(wilderDomainHash);
+    expect(resolvedAddress).to.equal(user.address);
   });
 
   it("Should allow operator to setAddress and emit event", async () => {
-    await znsRegistry.connect(owner).setOwnerOperator(operator.address, true);
+    await zns.registry.connect(deployer).setOwnerOperator(operator.address, true);
 
     await expect(
-      znsAddressResolver.connect(operator)
-        .setAddress(wilderDomainNameHash, addr1.address)
+      zns.addressResolver.connect(operator)
+        .setAddress(wilderDomainHash, user.address)
     )
-      .to.emit(znsAddressResolver, "AddressSet")
-      .withArgs(wilderDomainNameHash, addr1.address);
+      .to.emit(zns.addressResolver, "AddressSet")
+      .withArgs(wilderDomainHash, user.address);
   });
 
   it("Should allow REGISTRAR_ROLE to setAddress and emit event", async () => {
-    await accessController.connect(deployer).grantRole(REGISTRAR_ROLE, mockRegistrar.address);
+    await zns.accessController.connect(deployer).grantRole(REGISTRAR_ROLE, mockRegistrar.address);
 
     await expect(
-      znsAddressResolver.connect(mockRegistrar)
-        .setAddress(wilderDomainNameHash, addr1.address)
-    )
-      .to.emit(znsAddressResolver, "AddressSet")
-      .withArgs(wilderDomainNameHash, addr1.address);
-  });
+      zns.addressResolver.connect(mockRegistrar).setAddress(wilderDomainHash, hre.ethers.constants.AddressZero)
+    ).to.emit(zns.addressResolver, "AddressSet").withArgs(wilderDomainHash, hre.ethers.constants.AddressZero);

```

Comment:
again here. this looks dirty and hard to read. why did you reformat? how does this help?